frontend/src/components/MyForge/tabs/UseAMachine.tsx

```
@@ -135,6 +138,17 @@ export const DynamicMachineForm: React.FC = () => {
         setResourceUsageForm(newResourceUsageForm);
     }, [schema]);
 
+    function updateErrorText(message: string) {
+        setErrorText(message);
+        setErrorType("error");
+        try { document.getElementById('error-text')?.scrollIntoView({ behavior: 'smooth', block: 'center' }); } catch {}
+    }
+
+    function updateSuccessText(message: string) {
+        setErrorText(message);
+        setErrorType("success");
+        try { document.getElementById('error-text')?.scrollIntoView({ behavior: 'smooth', block: 'center' }); } catch {}
```

Empty catch block is spooky. At least log an error to console so this can be debugged in future.